static/styles/zulip.scss

```diff
@@ -1244,20 +1244,49 @@ div.focused_table {
     }
 }
 
-#inline_topic_edit,
-#message_edit_topic {
+#inline_topic_edit {
     margin-bottom: 5px;
 }
 
+#message_edit_topic {
+    margin: 0 5px 5px 0;
+}
+
+.message_edit_header {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+}
+
 .select_edit_stream {
     margin-bottom: 5px !important;
+    border-left: 0px;
+    padding-left: 0px;
+    border-radius: 0px 3px 3px 0px;
+    margin-left: -11px;
+    text-indent: 10px;
+}
+
+#stream_header_colorblock {
+    /* box-shadow: 0px 2px 3px hsl(0, 0%, 80%); */
+    box-shadow: inset 0px 2px 1px -2px hsl(0, 0%, 20%), inset 2px 0px 1px -2px hsl(0, 0%, 20%);
+    width: 10px;
+    border-radius: 3px 0px 0px 3px;
+    border: 1px solid hsla(0, 0%, 0%, 0.2);
+    border-right: 0px;
+    box-shadow: none;
+    display: inline-block;
+    vertical-align: middle;
+    margin-bottom: 5px;
+    z-index: 1;
```

Comment:
Can you extract a `.stream_header_colorblock` CSS class for the visual styling of these blocks, in `app_components.scss`, and use that in all 3 contexts (compose, popover, inline message edit)?  We could do that extraction as an initial prep commit.  

Duplicate IDs are problematic for selectors, and will be present since one can have multiple `message_edit` widgets open.  And this is a lot of duplicated visual styling CSS.

And then we can access these elements in JS code via $(selector_for_container).find(".stream_header_colorblock")

Reply:
I modified the code to use same classes in popover and message_edit. The compose stream colobar bar css is a bit different so, I left it to be separate.